mbrl/models/gaussian_mlp.py

```diff
@@ -81,15 +81,18 @@ def __init__(
         deterministic: bool = False,
         propagation_method: Optional[str] = None,
         learn_logvar_bounds: bool = False,
+        activation = relu,
```

Comment:
I have used `activation_name` for better readability and distinction between activation class and name.

Reply:
Agreed on `activation_name`. On the other hand, while we will certainly remove `use_silu` eventually, this is a change that breaks existing functionality, since our current `hydra` configurations won't work anymore. So, rather than removing it, we should emit a deprecation warning saying that this argument has now been replaced by `activation_name`. See [this](https://github.com/facebookresearch/mbrl-lib/blob/fa3f9b922db9b61d81cafb359546c5bb78440cd6/mbrl/util/replay_buffer.py#L538) for an example of how we have handled this in the past.